rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -768,7 +768,10 @@ namespace rsx
 
 		method_registers[NV4097_SET_LINE_WIDTH] = 1 << 3;
 
-		method_registers[NV4097_SET_FOG_MODE] = 0x0800; // rsx::fog_mode::exponential;
+		// These defaults were found using After Burner Climax (which never set fog mode despite using fog input)
+		method_registers[NV4097_SET_FOG_MODE] = 0x2601; // rsx::fog_mode::linear;
+		(f32&)method_registers[NV4097_SET_FOG_MODE] = 1.;
```

Comment:
After Burner never touch the NV4097_SET_FOG_MODE/NV4097_SET_FOG_PARAMS.
However it uses the fogc input in some fragment shader and according to some youtube video there is indeed fog in the long distance.

Linear/1./1. values leaves the fogc input unmodified (ie fogc = 1. * In.fogc + 1. - 1.) and the visual matches the video youtube.

BTW I fixed the typo.